javacord-core/src/main/java/org/javacord/core/entity/user/MemberImpl.java

```diff
@@ -300,6 +312,11 @@ public boolean isPending() {
         return pending;
     }
 
+    @Override
+    public Optional<Instant> getTimeout() {
+        return Optional.ofNullable(communicationDisabledUntil);
```

Comment:
If the timeout is expired, the returned optional should be empty
```suggestion
        return Optional.ofNullable(communicationDisabledUntil).filter(Instant.now()::isBefore);
```

Reply:
I see, that's a legit case too. Maybe two functions? `getTimeout()` and `getActiveTimeout()`?